google/resource_compute_subnetwork.go

```diff
@@ -117,7 +117,6 @@ func resourceComputeSubnetwork() *schema.Resource {
 			},
 			"secondary_ip_range": {
 				Type:     schema.TypeList,
-				Computed: true,
```

Comment:
Hi @rileykarson, removing this caused https://github.com/terraform-providers/terraform-provider-google/issues/3477